### DISCUSSION TITLE
Qtile Wayland Dependencies

### DISCUSSION THREAD

**Discussion by cr1ogen** [qtile/qtile#4462] [Q&A] [answered]
A question. Is it necessary in Qtile to have wlroots and pywlroots installed, for it to work in Wayland? If I'm not mistaken, doesn't Sway work only with wlroots? Thank you.

**Comment by m-col** [comment, marked as answer]
Do you mean do you need wlroots _and_ pywlroots, rather than just wlroots?

If so, yes - wlroots is only a C library, and pywlroots is just a light python wrapper that lets us access wlroots C definitions from Qtile's python.

**Reply by cr1ogen** [reply]
Yes, I meant that, thank you very much.
